### PR TITLE
Data Editor UI has togglable color mode.

### LINKED ISSUE #536: Display themes don't change Data Editor background colors
Light themes are especially hard to read.

3acb2279-0784-4a5e-ad15-bf3e01177ab8

### PR DESCRIPTION
**Pull request by stricklandrbls**
- UI color theme pulls from user selected VSCode color theme.
- Some UI components have locked theme background to ensure readability across various themes. 

Closes #536

### CODE REVIEW

File: src/svelte/src/components/dataEditor.svelte [hunk 1]
```diff
@@ -774,658 +781,713 @@ limitations under the License.
...
-          >
-          <div class="col-item" id="disk_file_size">{$diskFileSize}</div>
-        </div>
-        <div class="two-row-item flex-container col">
-          <label class="col-item file-metrics" for="computed_file_size"
-            >Computed Size</label
-          >
-          <div class="col-item" id="computed_file_size">
-            {$computedFileSize}
+<body class={$UIThemeCSSClass}>
```
Comment (scholarsmate) [inline src/svelte/src/components/dataEditor.svelte:792]: Why use a `body` tag?  This is going to be injected into VS Code markup which will already have a body.

**Review by scholarsmate — COMMENTED** [review]
This update looks pretty nice.  Just a concern about using a `body` tag.

File: src/svelte/src/components/dataEditor.svelte (hunk 1 above)
Comment (stricklandrbls) [reply src/svelte/src/components/dataEditor.svelte:792]: I needed a way to set the global font colors per scheme but didn't have a way to inject any styling in the `SvelteWebviewInitializer` that contains the UI body. This was the quickest solution that I predict will change with the implementation of https://github.com/ctc-oss/daffodil-vscode/issues/110

**Review by Shanedell — APPROVED** [review]
+1 👍

**Pull request merged**